.buildkite/scripts/deploy.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+TERRAFORM_ROLE="arn:aws:iam::760097843905:role/platform-ci"
+
+# Do this here rather than in Dockerfile so we don't install it on every build/test
+apt install -y awscli zip
+ROOT=$(git rev-parse --show-toplevel)
+
+cd $ROOT/terraform
+terraform init -backend-config="role_arn=${TERRAFORM_ROLE}"
+S3_BUCKET=$(terraform output -raw s3_bucket)
+S3_KEY=$(terraform output -raw s3_key)
+FUNCTION_NAME=$(terraform output -raw function_name)
+cd $ROOT
+
+yarn package
+aws s3 cp $ROOT/package.zip "s3://${S3_BUCKET}/${S3_KEY}"
+VERSION_ID=$(aws s3api list-object-versions --bucket $S3_BUCKET --prefix $S3_KEY --query 'Versions[?IsLatest].[VersionId]' --output text)
```

Comment:
I think you can do line breaks inside `$(…)`? Let's add some for readability:

```suggestion
VERSION_ID=$(aws s3api list-object-versions \
  --bucket $S3_BUCKET \
  --prefix $S3_KEY \
  --query 'Versions[?IsLatest].[VersionId]' \
  --output text)
```